internal/config/benchmark.go

```diff
@@ -105,6 +109,22 @@ type RemoveConfig struct {
 	Timestamp            string `json:"timestamp,omitempty"`
 }
 
+// ObjectConfig defines the desired state of object config
+type ObjectConfig struct {
+	FilterConfig FilterConfig `json:"filter_config,omitempty" yaml:"filter_config"`
+}
+
+// FilterTarget defines the desired state of filter target
+type FilterTarget struct {
+	Host string `json:"host,omitempty" yaml:"host"`
+	Port int32  `json:"port,omitempty" yaml:"port"`
+}
+
+// FilterConfig defines the desired state of filter config
+type FilterConfig struct {
+	Targets []*FilterTarget `json:"target,omitempty" yaml:"target"`
+}
```

Comment:
I am not sure, is it ok to implement the `Bind` method to this config object (ObjectConfig, FilterTarget, FilterConfig)?

Reply:
I'll check it and fix if needed.